zboxcore/sdk/writemarker_mutex.go

```diff
@@ -308,21 +309,15 @@ func (wmMu *WriteMarkerMutex) lockBlobber(
 				if wmLockRes.Status == WMLockStatusPending {
 					logger.Logger.Info("Lock pending for blobber ",
 						b.Baseurl, "with connection id: ", connID, " Retrying again")
-					time.Sleep(timeOut * 2)
+					time.Sleep(WMLockWaitTime)
 					shouldContinue = true
+					retry--
 					return
 				}
 				err = fmt.Errorf("Lock acquiring failed")
 				return
 			}
 
-			if resp.StatusCode == http.StatusAccepted { // accepted but pending
```

Comment:
We should keep the code for processing the `StatusAccepted`, we need to wait and retry, otherwise, we will try to read body from the response and return error, which is not what expected.

Reply:
@peterlimg Does the blobber return StatusAccepted ever? I saw the blobber code and couldn't find it. Can you point to the file?